DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/CourseDelegates/CourseDelegatesViewModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.CourseDelegates
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.CourseDelegates;
+    using DigitalLearningSolutions.Web.Helpers;
+    using Microsoft.AspNetCore.Mvc.Rendering;
+
+    public class CourseDelegatesViewModel
+    {
+        public CourseDelegatesViewModel(CourseDelegatesData courseDelegatesData)
+        {
+            CustomisationId = courseDelegatesData.CustomisationId;
+            var courseOptions = courseDelegatesData.Courses.Select(c => (c.CustomisationId, c.CourseName));
+            Courses = SelectListHelper.MapOptionsToSelectListItems(courseOptions, CustomisationId);
+
+            Active = courseDelegatesData.Courses.SingleOrDefault(c => c.CustomisationId == CustomisationId)?.Active;
+
+            // TODO: HEEDLS-564 - paginate properly instead of taking 10.
+            Delegates = courseDelegatesData.Delegates.Take(10).Select(cd => new SearchableCourseDelegateViewModel(cd));
+        }
+
+        public int? CustomisationId { get; set; }
+        public IEnumerable<SelectListItem> Courses { get; set; }
+        public bool? Active { get; set; }
+
+        public IEnumerable<SearchableCourseDelegateViewModel> Delegates { get; set; }

```

Comment:
It might be neater to have customisationId, Active, and Delegates live on the same nullable object, e.g. SelectedCourseDetails or something - then if there are no customisations to choose from, the data is simply null but we're forcing the CustomisationId and Active to be populated if a course does exist.

Reply:
CustomisationId needs to stay on the base model as it breaks model binding otherwise